index.js

```diff
@@ -10,27 +10,176 @@ const profileList = webix.ui({
         webix.message({text: "Check fields please", type: "error"})
     }
 };
 
 const clearForm = function () {
+    const filmForm = $$("form-film");
     webix.confirm({
         text: "Are you sure?"
     }).then(function (result) {
         if (result) {
-            $$("form-film").clear();
-            $$("form-film").clearValidation();
+            filmForm.clear();
+            filmForm.clearValidation();
         }
     })
 };
 
+const tableInMainPart = {
+    view: "datatable",
+    id: "table-films",
+    autoConfig: true,
+    select: true,
+    url: "data/data.js",
+    gravity: 4,
+    scrollX: false,
+    hover: "hover-background",
+    columns: [
+        {id: "id", autoWidth: true, header:"", css: "row-in-table-background", width:60},
+        {id: "title", sort: "string", header: ["Title", {content: "textFilter"}], fillspace: true},
+        {id: "year", sort: "int", header: ["Year", {content: "textFilter"}]},
+        {id: "votes", sort: "int", header: ["Votes", {content: "textFilter"}]},
+        {id: "del", header:"", template: "{common.trashIcon()}", width:50}
+    ],
+    on: {
+        onAfterSelect: function (id) {
+            const film = this.getItem(id);
+            const filmForm = $$("form-film");
+            filmForm.clearValidation();
+            filmForm.setValues(film);
+        }
+    },
+    onClick: {
+        "wxi-trash": function (e, id) {
+            this.remove(id);
+            return false;
+        }
+    },
+    scheme: {
+        $init: function(obj){
+            if(obj.votes.includes(",")){
+                obj.votes = Math.ceil(obj.votes.replace(",",".")*1000);
+            }
+        }
+    }
+};
+
+const formInMainPart = {
+    view: "form",
+    id: "form-film",
+    gravity: 2.5,
+    elements: [
+        {type: "section", template: "Edit films"},
+        {view: "text", label: "Title", name: "title", invalidMessage: "Title must be filled in"},
+        {view: "text", label: "Year", name: "year", invalidMessage: "Year should be between 1900 and current"},
+        {view: "text", label: "Rating", name: "rating", invalidMessage: "Rating cannot be empty or 0"},
+        {view: "text", label: "Votes", name: "votes", invalidMessage: "Votes must be less than 1'000'000 and filled in"},
+        {
+            margin: 50, cols: [
+                {view: "button", value: "Add", css: "webix_primary", click: addFilm},
+                {view: "button", value: "Clear", click: clearForm}
+            ]
+        },
+        {}
+    ],
+    rules: {
+        title: webix.rules.isNotEmpty,
+        year: function (value) {
+            return 1900 <= value && value <= new Date().getFullYear();
+        },
+        rating: function (value) {
+            return webix.rules.isNotEmpty(value) && value !== "0"
+        },
+        votes: function (value) {
+            return webix.rules.isNotEmpty(value) && value < 1000000;
+        }
+    }
+};
+
+let xhr = webix.ajax().sync().get("data/users.js");
```

Comment:
it's better to avoid sync requests. 
also, these's no need to load data by one request to parse the same data into list and chart. by the task, they share the same data, but they can simply issue their one request for this data `url:"/data.users.js"`. 

if you want to use only one request, you can do this after layout inie ( `webix.ui` call ): 

~~~
webix.ajax("data/data.json", then:function(data){
    data = data.json();
    list.parse(data);
    chart.parse(data);
});
~~~